README.md

```diff
@@ -429,7 +301,7 @@ The following workarounds may be used instead of the "Add" button to help you ad
     - Run Procedure
     - Publish Application
     - Create and Deploy Application from Deployment Package
-- Updated Deploy application by possibility to specify project for environment if it is othe than project for application
+- Updated Deploy application by possibility to specify project for environment if it is other than project for application
```

Comment:
May consider rewording

Reply:
Won't rework release notes.

Reply:
@jsmith-cb okay, we'll want to fix the typo the next time we update release notes